### ISSUE TITLE
Nuke image customization

### ISSUE THREAD

**Issue by Midnayt** [roubachof/Xamarin.Forms.Nuke#21]
There is no way to customize Nuke handler and use all features from native library.

**Comment by roubachof** [comment]
it makes no sense since Nuke is only a ImageSourceHandler for XF. 
So the api is the same than xf :)

**Comment by Midnayt** [comment]
> it makes no sense since Nuke is only a ImageSourceHandler for XF.
> So the api is the same than xf :)

Glidex is ImageSourceHandler too, but there is possible to add custom handler.

**Comment by roubachof** [comment]
I'm afraid I don't follow you... glidex is also just an implementation of IImageSourceHandler. 

glidex.forms/ImageViewHandler.cs

**Comment by Midnayt** [comment]
> I'm afraid I don't follow you... glidex is also just an implementation of IImageSourceHandler.
> 
> glidex.forms/ImageViewHandler.cs

Yes, but Glidex has GlideExtensions and IGlideHandler which allow make own customization.
Sample
https://github.com/jonathanpeppers/glidex/blob/master/glidex.forms.sample/RandomAlphaHandler.cs

**Comment by roubachof** [comment]
I see. But this example is very glide specific based on request.
What type of interface would you need ?
Please make a spec.
